cmd/common.go

```diff
@@ -566,6 +588,20 @@ func getUserId(c *api.Client) (string, error) {
 	return u.ID, nil
 }
 
+func getWorkspaceOrDefault(c *api.Client) (string, error) {
```

Comment:
Went with removing it because I would have had to update all instances that used `viper.GetString(WORKSPACE)` to use `getWorkspaceOrDefault` if the "name as id" equivalent was set. Seemed more hassle than what it is worth